docs/guide/docs/tutorials/hitl.md

```diff
@@ -52,3 +54,155 @@ The agent can then navigate to the appropriate conversation and take over the co
+
+To set any of those configuration, you'll first need to open up the `hitlnext.json` in the code editor section of your Botpress Studio.
+
+![Module Configuration](assets/hitl/hitl-config.png)
+
+#### Transfer and assignation messages
+
+Say we want to change **Transfer message** (default being `You are being transferred to an agent`) and want to add a German translation for your German speaking users. Simply change the `transferMessage` json object to the desired translations.
+
+```
+"transferMessage": {
+  "en": "I'm tranfering you to our support team",
+  "de": "Ich schicke dich zur Unterstützung"
+}
+```
+
+To see these changes, you'll need to either disable your chatbot and re-enable it or to restart your Botpress server.
+
+Same technique applies to **Assignation Message**, the only difference is that you have access to the `agentName` variable which corresponds to the agent full name. Using this variable is optional, refer to the default `assignMessage` configuration for an example usage.
+
+#### Message templates
+
+If your team of agents often use the same set of answers, you can use the `autoComplete` configuration to set pre-written messages so they don't have to go and type them over and over again. You can view them as shortcut for agents, none are defined by default and you can add as many as you want. Here is how they show up in the Agent Interface :
+
+![Shortcuts](assets/hitl/shortcuts.png)
+
+Before adding any message template, you need to choose a `trigger` character that will open up the templates list, "**:**" in this example. Then, for each of you messages you need to add a `name` (what you see on the left of each template and can be used to filter down the list as the agent types) and a `value` (the actual message that is going to be pasted in the composer). To summarize, configuration for the example above would look like the following:
+
+```
+"autoComplete": {
+  "trigger": ":",
+  "shortcuts": [
+    {
+      "name": "hi",
+      "value": "Hello you, I'm your kind agent, here to help you with whatever questions you might have."
+    },
+    {
+      "name": "qa",
+      "value": "Did I answer properly to all of your questions, is there anything else I can help you with today?"
+    },
+    {
+      "name": "bye",
+      "value": "Than you for trusting us, have a wonderful day."
+    }
+  ]
+}
+```
+
+#### Handoff Labels
+
+Call them tags, labels or categories, they are useful for your agents to categorize handoffs prior resolution so future agents can quickly know what a conversation is about. Tags also come in handy when doing an overview of resolved handoffs either for quality assurance or for reporting. You can add as many tags as you want but a good practice is to start with a small number of tags and adding new when necessary. Here is how they show up in the agent interface:
```

Comment:
We should use only one term, to avoid confusion. Either tags, labels or categories, but we need to choose one. This helps us have a shared vocabulary with our clients.

Reply:
I'll go for Labels even though the config is being the tags key